packages/core/admin/admin/src/pages/Settings/components/SettingsNav.tsx

```diff
@@ -5,19 +5,23 @@ import {
   SubNavSection,
   SubNavSections,
 } from '@strapi/design-system';
-import { Lock } from '@strapi/icons';
+import { Lightning } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 import { NavLink, useLocation } from 'react-router-dom';
 import { styled } from 'styled-components';
 
 import { useTracking } from '../../../features/Tracking';
 import { SettingsMenu } from '../../../hooks/useSettingsMenu';
 
-const CustomIcon = styled(Lock)`
+const CustomIcon = styled(Lightning)`
   right: 15px;
   position: absolute;
   bottom: 50%;
   transform: translateY(50%);
+
+  path {
+    fill: ${({ theme }) => theme.colors.warning500};
+  }
```

Comment:
```suggestion
fill: ${({ theme }) => theme.colors.warning500};
```

nitpick, but with new SVGs you can just apply a normal fill, no need to target the path.

Reply:
without the path I lose the specificity that I need
![Schermata 2024-06-11 alle 16 20 50](https://github.com/strapi/strapi/assets/2589748/138763e8-e606-4e14-988f-15d3fa797e26)
If it is ok for you I am gonna stay with my style